pkg/controller.v1/tensorflow/controller.go

```diff
@@ -55,6 +55,8 @@ const (
 	labelGroupName      = "group-name"
 	// Deprecated label for backwards compatibility. Has to be removed
 	labelTFJobName = "tf-job-name"
+	// volcanoTaskSpecKey task spec key used in pod annotation when EnableGangScheduling is true
+	volcanoTaskSpecKey = "volcano.sh/task-spec"
```

Comment:
cc @denkensk

Reply:
@jiangkaihua  I suggest this is better to add the annotation like scheduling.k8s.io/task-spec=ps to announce task type.
This will be consistent with scheduling.k8s.io/group-name. This will make the naming more general and convenient for users. `task-spec:ps/trainer` is the proper name of distributed training. A general name is more reasonable other than `volcano task spec`.
/cc @gaocegege

Reply:
Should we discuss it with volcano members?

Reply:
Yes, l think it makes sense, but it needs some modification to volcano. 
/cc @jiangkaihua

Reply:
Is there any other project may use this annotation?

Reply:
Yes, Our project will also need to know the `task-spec` of the pod. More important is that  `task-spec` is the proper name of distributed training. A general name is more reasonable.